packages/prices-api/src/savings/models.ts

```diff
@@ -31,9 +26,13 @@ export type Revenue = {
   timestamp: Date
 }
 
-export type Statistics = {
-  lastUpdated: Date
-  lastUpdatedBlock: number
-  aprProjected: number
-  supply: number
+export type Statistics = { lastUpdated: Date; lastUpdatedBlock: number; aprProjected: number; supply: number }
+
+export type UserStats = {
+  totalDeposited: string
+  totalReceived: string
+  totalWithdrawn: string
+  totalTransferredIn: string
+  totalTransferredOut: string
+  currentBalance: string
```

Comment:
The API client is supposed to convert these

```suggestion
  totalDeposited: number
  totalReceived: number
  totalWithdrawn: number
  totalTransferredIn: number
  totalTransferredOut: number
  currentBalance: number
```